src/main/java/com/code_intelligence/jazzer/driver/OptParser.java

```diff
@@ -70,7 +70,7 @@ static String stringSetting(String name, String defaultValue, String description
   }
 
   static List<String> stringListSetting(String name, String description) {
-    return stringListSetting(name, File.pathSeparatorChar, description);
+    return stringListSetting(name, ',', description);
```

Comment:
Let's keep this breaking change out of this PR and have a separate one for it. I do think we should make it, but we have to think about the implications for users.